rotkehlchen/tests/api/test_misc.py

```diff
@@ -409,3 +409,85 @@ def test_query_all_chain_ids(rotkehlchen_api_server):
     response = requests.get(api_url_for(rotkehlchen_api_server, 'allevmchainsresource'))
     result = assert_proper_response_with_result(response)
     assert result == [{'id': chain.value, 'name': chain.to_name()} for chain in ChainID]
+
+
+def test_images_cache_deletion(rotkehlchen_api_server):
```

Comment:
Set a temporary directory for this test since you are going to be creating and deleting files in that directory. 
```suggestion
@pytest.mark.parametrize('use_clean_caching_directory', [True])
def test_images_cache_deletion(rotkehlchen_api_server):
```